jenkins/toggle-webhook-processor.py

```diff
@@ -0,0 +1,63 @@
+import boto3
+import sys
+import logging
+
+
+def main():
+    try:
+        # connect to aws lambda
+        client = boto3.client('lambda')
+    except Exception, e:
+        # if connection failed, log error
+        logging.error(e)
+        sys.exit(1)
+
+    # ensure the number of arguments is correct
+    num_args = len(sys.argv)
+
+    if num_args == 2:
+        # parse toggle argument
+        arg = sys.argv[1]
+
+        try:
+            # find event source mapping for webhook_processor
+            response = client.list_event_source_mappings(
+                FunctionName='webhook_processor'
+            )
+            uuid = response["EventSourceMappings"][0]["UUID"]
+        except:
+            # if none found, lambda does not exist
+            logging.error(
+                "No EventSourceMappings found for that lambda function name"
+            )
+            sys.exit(1)
+
+        # call method according to the passed argument
+        if arg == "ON":
+            toggle(client, uuid, True)
+        elif arg == "OFF":
+            toggle(client, uuid, False)
+        else:
+            logging.error("Invalid argument. Please specify ON or OFF")
+    else:
```

Comment:
The code will be more readable if you flip the "if" and "else" conditions. This would be the "if" and there would be no need for the rest of the code to be nested in an "else".
(This is a moot point if you switch to using an argument parser.)